### DISCUSSION TITLE
Just made new logo for Z#. I think quite a result!

### DISCUSSION THREAD

**Discussion by VyaCHACHsel** [sam-astro/Z-Sharp#27] [Show and tell]
![z#](https://user-images.githubusercontent.com/60910714/170837582-8a59c339-4a92-4db7-aca9-2ae46b67d41d.png)
also version with text:
![z#-full](https://user-images.githubusercontent.com/60910714/170837823-895e3251-0106-4ef1-bd05-ef6bf6d7ad94.png)
more different versions (like monochrome, flat or 3-colored version)
![Z# logo](https://user-images.githubusercontent.com/60910714/170837867-189089ae-6fda-4c8a-ae3e-5ed92a26d02d.png)

What do you think?

**Comment by sam-astro** [comment]
These are incredibly cool, mine was just a copy of the traditional C ones, but you truly made something original! It's too bad the project isn't very big and I'm not continuing on it, because I would love to use these. Continue being creative and thank you for your contribution! 😁

**Reply by VyaCHACHsel** [reply]
It just went into my head spontaneously to be honest. I can do a pull request with this new logo if you want to. Maybe one day it will become one of great high-level languages. (and yes I know you don't really update it anymore, but maybe someone other can take this?)

**Comment by bsikar** [comment]
It feels really close to Haskell and Z-Sharp isnt anything like Haskell, but still cool (I like the bottom left red one)
for reference: [Haskell Logo](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fchrisconlan.com%2Fwp-content%2Fuploads%2F2018%2F06%2Fhaskell_logo_2.png&f=1&nofb=1)
![image](https://user-images.githubusercontent.com/65072072/172672493-2022ad06-beb9-4ec8-a92e-1e774d37261c.png)

**Reply by VyaCHACHsel** [reply]
Woah! So Haskel already got something similar...

Oh well.
